renderer/components/dashboard-projects/dashboard-filter/dashboard-filter.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ChangeEvent, MutableRefObject } from 'react';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import { technologies } from './technologies-list';
+
+interface DashboardFilterProps {
+  value: string;
+  filterRef: MutableRefObject<HTMLInputElement>;
+  filterHandler: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const DashboardFilter = (props: DashboardFilterProps): JSX.Element => {
+  const onChangeFilterHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    props.filterHandler(event);
+  };
+  return (
+    <FormControl variant="outlined">
+      <TextField
+        select
+        value={props.value}
+        variant="outlined"
+        inputRef={props.filterRef}
+        onChange={onChangeFilterHandler}
+        label="Filter"
+        inputProps={{
+          id: 'filter',
+        }}
+      >
+        <MenuItem value={'date'}>By Date Created</MenuItem>
+        <MenuItem value={'name'}>By Name</MenuItem>
+        <MenuItem
+          value={''}
+          disabled

```

Comment:
Why is this item disabled?

Reply:
According to the design, By technology is not clickable.

![image](https://user-images.githubusercontent.com/29056094/91689694-44287780-eb82-11ea-9776-b4dc04cee98c.png)

Reply:
I thought you could click on "By technology" and then write it on "Search", but also click on "React" or "Eclipse" so you don't have to write it. Please check it with Yadav

Reply:
As discussed with Yadav last week, I have updated it.  

If we click any technology like node, java, angular, first needs to filter with selected technology. And search input also there. If we enter a character, then we need to filter again on top of filtered technology.